index.html

```diff
@@ -1,40 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link rel="stylesheet" href="./Styles/styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
-    />
-  </head>
-  <body>
-    <header>
-      <a href="index.html" class="logo"><h1>Byiringiro Moise</h1></a>
-      <span class="material-symbols-outlined"> menu </span>
-    </header>
-    <section class="profile">
-        <div class="introduction">
-<h2>Hey There. I'm Moise</h2>
-<h3>I'am a Software Developer</h3>
-<p>
-    I can help you build a product, feature or website look through some of my work and experience!
-    if you like what you see and have a project you need coded, don't hesitate to contact me.
-</p>
-        </div>
-        <div>
-          <ul class="social-media-icons">
-<li><a href=""><img src="/items/Github-icon.png" alt="github icon"></a></li>
-<li><a href=""><img src="/items/linkedin-icon.png" alt=""></a></li>
-<li><a href=""><img src="/items/Icon - Angelist.png" alt=""></a></li>
-<li><a href=""><img src="/items/Icon - Twitter.png" alt=""></a></li>
-<li><a href=""><img src="/items/M-icon.png" alt=""></a></li>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="./Styles/styles.css" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+
+<body>
+  <header>
+    <a href="index.html" class="logo">
+      <h1>Byiringiro Moise</h1>
+    </a>
+    <span class="material-symbols-outlined"> menu </span>
+  </header>
+  <section class="profile">
+    <div class="introduction">
+      <h2>Hey There. I'm Moise</h2>
+      <h3>I'am a Software Developer</h3>
+      <p>
+        I can help you build a product, feature or website look through some of my work and experience!
+        if you like what you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+    </div>
+    <div>
+      <ul class="social-media-icons">
+        <li><a href=""><img src="/items/Github-icon.png" alt="github icon"></a></li>
+        <li><a href=""><img src="/items/linkedin-icon.png" alt=""></a></li>
+        <li><a href=""><img src="/items/Icon - Angelist.png" alt=""></a></li>
+        <li><a href=""><img src="/items/Icon - Twitter.png" alt=""></a></li>
+        <li><a href=""><img src="/items/M-icon.png" alt=""></a></li>
+      </ul>
+    </div>
+    <img src="/items/Illustration (1).png" alt="" class="illustration">
```

Comment:
- Kindly make sure your body takes all screen without the scrolling button as shown in the picture below. to fix that i suggest you use just two background  images that contain all those images in the background and  add those background images in your CSS  to your about session and profile section instead of adding those background images in the HTML file  :+1: :grinning: :grinning: :grinning: 
- check below your design compared to the Figma design 
## your design
![image](https://user-images.githubusercontent.com/95297251/191872514-0e541fb6-1b5e-46b0-bea1-c561c9834999.png)

![image](https://user-images.githubusercontent.com/95297251/191872569-2380a991-ca02-4b37-836b-762ce9e5ffd7.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/191872738-f7554c3e-11ba-4a6f-bf53-624af30ce07e.png)

![image](https://user-images.githubusercontent.com/95297251/191872766-f77ffae5-a3af-4d19-8e55-007ad1e0276b.png)